src/bpmn-properties-panel/provider/bpmn/properties/ErrorProps.js

```diff
@@ -37,14 +38,26 @@ export function ErrorProps(props) {
   const error = getError(element);
 
   let entries = [
-    { id: 'errorRef', component: <ErrorRef element={ element } /> }
+    {
+      id: 'errorRef',
+      component: <ErrorRef element={ element } />,
+      isDirty: selectIsDirty
```

Comment:
I like the architecture of the solution and how flexible we can add the isDirty function